.localizationrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "paths": {
```

Comment:
__nit:__ maybe `.i18nrc.json` (a bit shorter and covers a bit broader scope that we may eventually need)?